packages/lit-element/src/test/lit-element_dev_mode_disable_test.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2018 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {LitElement} from 'lit-element';
+import {assert} from 'chai';
+
+// Note, since tests are not built with production support, detect DEV_MODE
+// by checking if warning API is available.
+const DEV_MODE = !!LitElement.enableWarning;
+
+if (DEV_MODE) {
+  globalThis.litIssuedWarnings ??= new Set();
+  globalThis.litIssuedWarnings!.add('dev-mode');
```

Comment:
dev-mode warning is now delayed by a microtask so we can disable the warning here (even after lit-element is already imported)

However, we can't do so inside of test/suiteSetup as they run later than a microtask.
Not an issue for my use case as disabling warnings in the global scope is now easy (so my library can do so for all consumers).